x-pack/test/functional_enterprise_search/README.md

```diff
@@ -0,0 +1,33 @@
+# Enterprise Search Functional E2E Tests
+
+## Running these tests
+
+Follow the [Functional Test Runner instructions](https://www.elastic.co/guide/en/kibana/current/development-functional-tests.html#_running_functional_tests).
+
+There are two suites available to run, a suite that requires a Kibana instance without an `enterpriseSearch.host`
+configured, and one that does. The later also [requires a running Enterprise Search instance](#enterprise-search-requirement), and a Private API key
+from that instance set in an Environment variable.
+
+Ex.
+
+```
+# Run specs from the x-pack directory
+cd x-pack
+
+# Run tests that require enterpriseSearch.host variable
+APP_SEARCH_API_KEY=private-2shg47w6zpj62zqe8qb5fap9 node scripts/functional_tests --config test/functional_enterprise_search/with_host_configured.config.ts
```

Comment:
Are we concerned at all about having this API key in a publicly-exposed document? I'm not sure if it's real or fake, but if fake, can we at least make it something like `private-12345`?

If it's meant to be a private key from the currently-spun-up local App Search instance, can we make that clearer via something like:

`APP_SEARCH_API_KEY=[use private key from local App Search instance here]`

Although I'm also actually not totally clear what we need the API key for?...

Reply:
OK after testing myself locally, looks like it's supposed to be a real private key from local AS. I'd suggest this:

```suggestion
APP_SEARCH_API_KEY=[use private key from local App Search instance here] node scripts/functional_tests --config test/functional_enterprise_search/with_host_configured.config.ts
```

Reply:
It looks like this change was also reverted at some point